styles/github/catppuccin.user.css

```diff
@@ -1502,6 +1561,73 @@
         background-color: @surface1;
     }
 
+    select option, optgroup {
+      background-color: @mantle;
+      border: 1px solid @surface1;
+      border-radius: 6px;
+      box-shadow: var(--shadow-floating-large, var(--color-shadow-large));
+       
+    }
+   
+    .hUQSCH {
+        background-color: @base;
+        color: @subtext1;
+        border-color: @surface1;
+       
+        &:hover {
+             background-color: @mantle;
+            border-color: @surface1;
+        }
+    }
+
+    /* Add File Menu */
+    .dQPLyc {
+        background-color: transparent;
+        border: 1px solid @surface1;
+    }
+  
+    .ktGGEf {
+        background-color: @base;
+        border: 1px solid @surface1;
+    }
+  
+    .gkfBtX {
+        background-color: @mantle;
+        border-radius: 6px 0px 0px 0px;
+        border-bottom: 1px solid @surface1;
+    }
+  
+    .jproXT {
+        background-color: transparent;
+        border-color: transparent;
+        color: @subtext1;
+      
+        &:hover {
+            color: @text;
+        }
+      
+        &:focus-within {
+            border-color: transparent;
+            box-shadow: none;
+            color: @text;
+        }
+    }
+  
+    .wLDmP {
+        background-color: @mantle;
+        border: 1px solid @surface1;
+    }
+  
+    .eNSiNz .segmentedControl-content, .gKuCoB .segmentedControl-content {
+        background-color: @base;
+    }
+
+    /* Notification Menu Fix */
+    .notifications-list-item.notification-read, .notifications-list-item.notification-archived {
+        background-color: @base !important;
+        border-radius: 6px
+    }
+
```

Comment:
![image](https://github.com/catppuccin/userstyles/assets/71222764/1c91aa82-dd93-4435-8927-d1c60dcc4289)
The notification hover curve looks a little strange.